src/Components/Button/Base/ButtonBase.tsx

```diff
@@ -85,6 +88,11 @@ export const ButtonBase = forwardRef<ButtonBaseRef, ButtonBaseProps>(
       }
     )
 
+    const titleTextToBeUsed =
+      status == ComponentStatus.Disabled && disabledTitleText
```

Comment:
Always prefer strictly equals when possible.
```suggestion
      status === ComponentStatus.Disabled && disabledTitleText
```